ansible/vagrant_site.yml

```diff
@@ -143,6 +143,9 @@
 
 - hosts: ronexserial
   sudo: yes
+  vars:
+    ros_release: 'indigo'
+    ros_user: 'ronexserial'
```

Comment:
@ugocupcic so you want to set ronexserial as default user for all usages of vagrant_site.yml. What about other installation do they use the same vagrant_site.yml or other files?